pkg/config/role_mappings.go

```diff
@@ -0,0 +1,36 @@
+package config
```

Comment:
I would move this to pkg/mappings/mappings.go and make it a struct on it's own.  Remove the distinction between Role/User mappings at this level.  They will still exist in the Config for backwards compatability until we remove them.  

```
struct Mappings {
  lock sync.RWMutex
  map[string]Mapping
}
```

Reply:
You also need to add a `Get` function as reading from a map is not safe to do concurrently.

Reply:
Awesome, thank you for the feedback - I'll get this updated. Thanks @mattlandis